### PR TITLE
Docker build

### PR DESCRIPTION
**Pull request by jhoenicke**
Use pyinstaller to build ModelValidator.
Add docker build support.

This also requires a small patch to the scrambler Makefile, so that the scrambler build works in docker: https://github.com/SMT-COMP/scrambler/commit/a84210dfb886ff0f80316a31830f9b8643038959

### CODE REVIEW

**Review by HanielB — APPROVED** [review]
Never used docker, so I trust you. :)

File: README.md [hunk 1]
```diff
@@ -9,3 +9,15 @@ This repository contains postprocessors for SMT-COMP.
 2. Create a tarball with the process wrapper script in the . directory of the tarball.
 
 SMT-COMP 2020 Model validator (experimental)
```
Comment (aehyvari) [inline README.md:11]: Can we update this year?

File: docker/create-docker.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+rm -rf scramblere postprocessors
```
Comment (aehyvari) [inline docker/create-docker.sh:3]: Is `scramblere` a typo?

**Review by aehyvari — COMMENTED** [review]
When I run `./build-docker.sh` I get an error 
```
tar -C unsat-core-track -czf SMT-COMP-2021-unsat-core-post-processor.tar.gz process scrambler validation_solvers/alt-ergo validation_solvers/bitwuzla validation_solvers/cvc4 validation_solvers/mathsat validation_solvers/ultimateeliminator+mathsat validation_solvers/vampire validation_solvers/yices validation_solvers/z3
make: *** No rule to make target `model-validation-track/requirements.txt', needed by `SMT-COMP-2021-model-validation-post-processor.tar.gz'.  Stop.
```

Is there maybe a missing file somewhere?

File: README.md (hunk 1 above)
Comment (jhoenicke) [reply README.md]: I rewrote the README completely.  No need to mention the year in the README

**Pull request merged**